packages/salesforcedx-utils-vscode/src/messages/i18n.ja.ts

```diff
@@ -41,6 +41,6 @@ export const messages = {
   channel_end: 'が終了しました。',
   predicates_no_folder_opened_text:
     'フォルダが開かれていません。VS Code で Salesforce DX 形式のプロジェクトを開いてください。',
-  predicates_no_sfdx_project_found_text:
+  predicates_no_sf_project_found_text:
     '開かれたプロジェクトのルートディレクトリに sfdx-project.json ファイルが見つかりませんでした。VS Code で Salesforce DX 形式のプロジェクトを開いてください。'
```

Comment:
Here the file name is still kept as `sfdx-project.json` instead of being switched to `sf-project.json` or `salesforce-project.json`.  (The name `sf-project.json` doesn't look too weird to me but if we're going to call them "Salesforce projects" instead of "SF projects" maybe we should be more clear with the naming so that we don't confuse our customers with that acronym?)